lib/Optimizer/CodeGen/LowerToQIRProfile.cpp

```diff
@@ -257,7 +257,7 @@ struct AddFuncAttribute : public OpRewritePattern<LLVM::LLVMFuncOp> {
   }
 
   const FunctionAnalysisInfo &infoMap;
-  llvm::StringRef convertTo;
+  std::string convertTo;
```

Comment:
I'm not entirely sure that I understand the benefit of doing this as it will require multiple `O(n)` conversions to take place when populating the various data structures containing `std::string convertTo` because we will have to call `StringRef.str()` when passes are constructed. (That being said - the strings are quite small, so it won't add a noticeable performance impact.) I made the changes in bc96b9a2 just to see if the ARM problems occur with these requested changes.

Reply:
The CI ran successfully with these changes (twice), so hopefully this change eliminates the need for a new issue.